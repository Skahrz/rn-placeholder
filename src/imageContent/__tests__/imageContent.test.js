import React from 'react';
import { shallow } from 'enzyme';
import ImageContent from '../imageContent';

describe('ImageContent', () => {
  let props;

  const getWrapper = () => shallow(<ImageContent {...props} />);

  beforeEach(() => {
    props = {
      position: 'left',
      size: 50,
      hasRadius: true,
      animate: 'fade',
      lineNumber: 8,
      textSize: 13,
      color: '#ff0000',
      width: '99%',
      lastLineWidth: '50%',
      firstLineWidth: '30%',
    };
  });

  it('should match snapshot', () => {
    expect(getWrapper()).toMatchSnapshot();
  });

  it('should match snapshot with right element', () => {
    props.position = 'right';
    expect(getWrapper()).toMatchSnapshot();
  });

  it('should match snapshot with left element', () => {
    props.position = 'left';
    expect(getWrapper()).toMatchSnapshot();
  });
});
