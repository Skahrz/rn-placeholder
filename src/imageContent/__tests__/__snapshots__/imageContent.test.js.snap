// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageContent render should match snapshot 1`] = `
<Paragraph
  animation="fade"
  color="#ff0000"
  firstLineWidth="30%"
  lastLineWidth="50%"
  lineNumber={8}
  renderLeft={[Function]}
  renderRight={[Function]}
  textSize={13}
  width="99%"
/>
`;
